[2025-06-25T21:24:12.595+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: northwind.carregamento_postgres manual__2025-06-25T21:23:32.326827+00:00 [queued]>
[2025-06-25T21:24:12.606+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: northwind.carregamento_postgres manual__2025-06-25T21:23:32.326827+00:00 [queued]>
[2025-06-25T21:24:12.606+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-25T21:24:12.622+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): carregamento_postgres> on 2025-06-25 21:23:32.326827+00:00
[2025-06-25T21:24:12.628+0000] {standard_task_runner.py:60} INFO - Started process 228 to run task
[2025-06-25T21:24:12.632+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'northwind', 'carregamento_postgres', 'manual__2025-06-25T21:23:32.326827+00:00', '--job-id', '207', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp9x59t5yi']
[2025-06-25T21:24:12.635+0000] {standard_task_runner.py:88} INFO - Job 207: Subtask carregamento_postgres
[2025-06-25T21:24:12.681+0000] {task_command.py:423} INFO - Running <TaskInstance: northwind.carregamento_postgres manual__2025-06-25T21:23:32.326827+00:00 [running]> on host 6afc1f3750fb
[2025-06-25T21:24:12.783+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lrbf@cin.ufpe.br' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='northwind' AIRFLOW_CTX_TASK_ID='carregamento_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-06-25T21:23:32.326827+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-25T21:23:32.326827+00:00'
[2025-06-25T21:24:12.855+0000] {docker.py:359} INFO - Starting docker container from image meltano:latest
[2025-06-25T21:24:10.695+0000] {docker.py:429} INFO - Running meltano install...
[2025-06-25T21:24:12.370+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.370032Z[0m [[32m[1minfo     [0m] [1mInstalling 7 plugins          [0m
[2025-06-25T21:24:12.394+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.394016Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-postgres'[0m
[2025-06-25T21:24:12.431+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.430227Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-csv'[0m
[2025-06-25T21:24:12.465+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.463861Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-csv-latest'[0m
[2025-06-25T21:24:12.496+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.495557Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-postgres'[0m
[2025-06-25T21:24:12.530+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.529735Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv'[0m
[2025-06-25T21:24:12.578+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.574699Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv-postgres'[0m
[2025-06-25T21:24:12.631+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:12.630311Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv-csv'[0m
[2025-06-25T21:24:14.587+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:14.586914Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-postgres'[0m
[2025-06-25T21:24:14.651+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:14.651089Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-postgres'[0m
[2025-06-25T21:24:19.588+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:19.587418Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-csv' [0m
[2025-06-25T21:24:19.777+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:19.776678Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-csv-latest'[0m
[2025-06-25T21:24:21.413+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:21.413015Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv' [0m
[2025-06-25T21:24:22.913+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:22.913033Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv-postgres'[0m
[2025-06-25T21:24:23.175+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:23.174332Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv-csv'[0m
[2m2025-06-25T21:24:23.174627Z[0m [[32m[1minfo     [0m] [1mInstalled 7/7 plugins         [0m
[2025-06-25T21:24:24.089+0000] {docker.py:429} INFO - Meltano install completed. Executing command: meltano el tap-csv-latest target-postgres
[2025-06-25T21:24:25.179+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:25.178900Z[0m [[32m[1minfo     [0m] [1mEnvironment 'dev' is active   [0m
[2025-06-25T21:24:25.614+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:25.613180Z[0m [[32m[1minfo     [0m] [1mRunning extract & load...     [0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m
[2025-06-25T21:24:26.473+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.471924Z[0m [[32m[1minfo     [0m] [1mNo CSV file definitions found.[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.612+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.611625Z[0m [[32m[1minfo     [0m] [1mtarget-postgres v0.4.3, Meltano SDK v0.47.2[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.613+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.612008Z[0m [[32m[1minfo     [0m] [1mSkipping parse of env var settings...[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.613+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.612399Z[0m [[32m[1minfo     [0m] [1mTraceback (most recent call last):[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.613+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.612677Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/bin/target-postgres", line 8, in <module>[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.613+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.612998Z[0m [[32m[1minfo     [0m] [1m    sys.exit(TargetPostgres.cli())[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.614+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.613335Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 1442, in __call__[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.614+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.613696Z[0m [[32m[1minfo     [0m] [1m    return self.main(*args, **kwargs)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.615+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.613980Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 1363, in main[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2m2025-06-25T21:24:26.614365Z[0m [[32m[1minfo     [0m] [1m    rv = self.invoke(ctx)     [0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.616+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.614708Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/singer_sdk/plugin_base.py", line 97, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.616+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.615239Z[0m [[32m[1minfo     [0m] [1m    return super().invoke(ctx)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.617+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.615654Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 1226, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.618+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.616102Z[0m [[32m[1minfo     [0m] [1m    return ctx.invoke(self.callback, **ctx.params)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.618+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.616659Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 794, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.618+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.617147Z[0m [[32m[1minfo     [0m] [1m    return callback(*args, **kwargs)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.619+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.617707Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/singer_sdk/target_base.py", line 583, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.619+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.618211Z[0m [[32m[1minfo     [0m] [1m    target = cls(             [0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.620+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.618649Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/target_postgres/target.py", line 67, in __init__[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2m2025-06-25T21:24:26.619216Z[0m [[32m[1minfo     [0m] [1m    and self.config.get("dialect+driver") is not None[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2m2025-06-25T21:24:26.619661Z[0m [[32m[1minfo     [0m] [1mAssertionError: Need either the sqlalchemy_url to be set or host, port, user, password, and dialect+driver to be set[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.740+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.739209Z[0m [[31m[1merror    [0m] [1mExtraction failed             [0m [36mcode[0m=[35m1[0m [36mmessage[0m=[35m'No CSV file definitions found.'[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m
[2m2025-06-25T21:24:26.739467Z[0m [[31m[1merror    [0m] [1mLoading failed                [0m [36mcode[0m=[35m1[0m [36mmessage[0m=[35m'AssertionError: Need either the sqlalchemy_url to be set or host, port, user, password, and dialect+driver to be set'[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m
[2025-06-25T21:24:26.740+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.739827Z[0m [[32m[1minfo     [0m] [1mELT could not be completed: Extractor and loader failed.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.740+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.740102Z[0m [[32m[1minfo     [0m] [1mFor more detailed log messages re-run the command using 'meltano --log-level=debug ...' CLI flag.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.741+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.740329Z[0m [[32m[1minfo     [0m] [1mNote that you can also check the generated log file at '/opt/meltano/.meltano/logs/elt/2025-06-25T212425--tap-csv-latest--target-postgres/57756db4-8372-4127-b8ec-457d1e91a07a/elt.log'.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.741+0000] {docker.py:429} INFO - [2m2025-06-25T21:24:26.740554Z[0m [[32m[1minfo     [0m] [1mFor more information on debugging and logging: https://docs.meltano.com/reference/command-line-interface#debugging[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m57756db4-8372-4127-b8ec-457d1e91a07a[0m [36mstate_id[0m=[35m2025-06-25T212425--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-25T21:24:26.758+0000] {docker.py:429} INFO - Need help fixing this problem? Visit http://melta.no/ for troubleshooting steps, or to
join our friendly Slack community.

ELT could not be completed: Extractor and loader failed.
For more detailed log messages re-run the command using 'meltano --log-level=debug ...' CLI flag.
Note that you can also check the generated log file at '/opt/meltano/.meltano/logs/elt/2025-06-25T212425--tap-csv-latest--target-postgres/57756db4-8372-4127-b8ec-457d1e91a07a/elt.log'.
For more information on debugging and logging: https://docs.meltano.com/reference/command-line-interface#debugging
[2025-06-25T21:24:27.926+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 502, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 376, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 437, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2025-06-25T21:24:27.930+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=northwind, task_id=carregamento_postgres, execution_date=20250625T212332, start_date=20250625T212412, end_date=20250625T212427
[2025-06-25T21:24:27.944+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 207 for task carregamento_postgres (Docker container failed: {'StatusCode': 1}; 228)
[2025-06-25T21:24:27.962+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-25T21:24:27.976+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
